packit_service/worker/handlers/distgit.py

```diff
@@ -460,6 +463,19 @@ def pre_check(self) -> bool:
                         f"configuration: {self.job_config.allowed_committers}."
                     )
                     return False
+        elif self.data.event_type in (PullRequestCommentPagureEvent.__name__,):
+            commenter = self.data.actor
+            project_url = self.data.event_dict["project_url"]
+            logger.debug(
+                f"Triggering downstream koji build through comment by: {commenter}"
+            )
+            if commenter not in self.job_config.allowed_committers:
```

Comment:
since this is an explicit comment, I think the commenter doesn't need to be allowed in the project configuration but only has a write access to the dist-git repo...but maybe we can discuss it on standup

Reply:
Ok, so if I understand it correctly: re-triggering a koji-build through a comment should be allowed to more developers.
Some of them, which may be not allowed to do a koji-build through propose-downstream are instead allowed to re-trigger someone else propose-downstream koji-builds, right?

Reply:
my thinking was that with the configurable `allowed_committers` and `allowed_pr_authors` we wanted to preserve the proven packager workflow, but if someone explicitly creates a comment to retrigger the build, it doesn't make that much sense to me to require this user to be allowed in the configuration...@lachmanfrantisek what do you think? should we discuss this in a meeting?

Reply:
the conclusion from the architecture discussion: we will only require the user who retriggers the koji build by a dist-git comment to be a packager